pkg/serverless/daemon/daemon.go

```diff
@@ -27,7 +27,7 @@ import (
 // ShutdownDelay is the amount of time we wait before shutting down the HTTP server
 // after we receive a Shutdown event. This allows time for the final log messages
 // to arrive from the Logs API.
-var ShutdownDelay time.Duration = 1 * time.Second
+var ShutdownDelay = 200 * time.Millisecond
```

Comment:
Just confirming, in the description you mentioned 400ms is the sweet spot. Why 200 here?

Reply:
ah typo, I meant 200. All the screenshots of self-monitoring & RCs are with 200

Reply:
Gotcha. And are we more likely to drop logs with this PR or is that something that you haven't noticed?

Reply:
Haven't noticed any dropped logs through self-monitoring, I benchmarked how long it actually takes to receive the report log, and 90+% of the time it's within single milliseconds.